1-exercises/A-reading-list/script.js

```diff
@@ -1,26 +1,46 @@
 function readingList(books) {
-  // Write your code here...
+  let content = document.getElementById("content");
+  let bookList = document.createElement("ul");
+  content.appendChild(bookList);
+
+  books.forEach((book) => {
+    let items = document.createElement("li");
+    bookList.appendChild(items);
+    let paragraphs = document.createElement("p");
+    paragraphs.innerHTML = `${book.title}: ${book.author}`;
+    items.appendChild(paragraphs);
+    let bookImage = document.createElement("img");
+    bookImage.src = book.bookCoverImage;
+    items.appendChild(bookImage);
+    if (book.alreadyRead === true) {
+      items.style.backgroundColor = "green";
```

Comment:
Do you know how you would apply a style class defined in the CSS file, to the element?